src/scenes/Wallet/index.js

```diff
@@ -1,25 +1,16 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { injectIntl, defineMessages, intlShape } from 'react-intl';
-import Paper from 'material-ui/Paper';
+import { injectIntl } from 'react-intl';
 import { withStyles } from 'material-ui/styles';
 
 import MyBalances from './components/Balances/index';
+import FunctionRow from './components/FunctionRow/index';
 import WalletHistory from './components/History/index';
 import appActions from '../../redux/App/actions';
 import { AppLocation } from '../../constants';
-import ImportantNote from '../../components/ImportantNote/index';
 import styles from './styles';
 
-const messages = defineMessages({
-  balanceInfo: {
-    id: 'balance.info',
-    defaultMessage: 'If you would like to encrypt, backup, or restore your wallet, please do this from the Qtum Wallet. In the application menu, there is a "Launch Qtum Wallet" option.',
-  },
-});
-
-
 @injectIntl
 @withStyles(styles, { withTheme: true })
```

Comment:
are we using `classes` here? If not please remove `@withStyles` and the `import withStyles`